### ISSUE TITLE
Request: CSS: separate declarations with 2 newlines

### ISSUE THREAD

**Issue by albell** [Glavin001/atom-beautify#1073]
### Actual Results

```
.myClass {
    color: #bada55;
}
.myOtherClass {
    color: #decca1;
}
```

### Expected Results

Beautified CSS code should have 2 newlines between declarations:

```
.myClass {
    color: #bada55;
}

.myOtherClass {
    color: #decca1;
}
```

An empty line between declarations is dictated by all modern CSS formatting guides that I know of, including:

Google:
https://google.github.io/styleguide/htmlcssguide.xml?showone=Rule_Separation

Github:
http://primercss.io/guidelines/#spacing

Wordpress:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/

AirBnb:
https://github.com/airbnb/css#formatting

Idiomatic CSS:
https://github.com/necolas/idiomatic-css#4-format

I suggest that this formatting be the default for beautified CSS and SCSS.

Thanks for an awesome tool!

**Comment by prettydiff** [comment]
Use the option "Newline between rules".  This is supported by both Pretty Diff and JS Beautify.

**Comment by albell** [comment]
Yes. I know, my point is that option should be turned on by default. It's the wrong initial setting, according to virtually every industry standard.

**Comment by prettydiff** [comment]
Reopening for consideration.

**Comment by Glavin001** [comment]
It looks like JS-Beautify's default is also `true` for `newline_between_rules`: https://github.com/beautify-web/js-beautify/blob/57b89fcde24199b18dcbcb247c158420d79039dc/js/lib/beautify-css.js#L76

@albell : I would accept and merge a Pull Request making the appropriate changes to https://github.com/Glavin001/atom-beautify/blob/master/src/languages/css.coffee#L48

Please be sure all tests pass in Travis CI. Once Travis CI build is passing, I will merge.
Thank you.
